packages/@aws-cdk/aws-pipes-targets-alpha/lib/eventBridge.ts

```diff
@@ -0,0 +1,145 @@
+import { IInputTransformation, IPipe, ITarget, TargetConfig } from '@aws-cdk/aws-pipes-alpha';
+import { IEventBus } from 'aws-cdk-lib/aws-events';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+
+/**
+ * EventBridge target properties.
+ */
+export interface EventBridgeTargetParameters {
+  /**
+   * The input transformation to apply to the message before sending it to the target.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetparameters.html#cfn-pipes-pipe-pipetargetparameters-inputtemplate
+   * @default none
```

Comment:
`none` is not a correct `IInputTransformation` value, same for the following properties
```suggestion
   * @default - none
```